src/views/design/ScriptsOverview/index.tsx

```diff
@@ -222,17 +223,26 @@ const ScriptsOverview = withStyles(styles)(
                                     {checkAuthorityGeneral(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE)
                                         ? (
                                             <Box display="flex" alignItems="center" flex="0 0 auto">
-                                                <Button
-                                                    variant="contained"
-                                                    color="secondary"
-                                                    size="small"
-                                                    startIcon={<AddRounded />}
-                                                    onClick={() => {
-                                                        redirectTo({ routeKey: ROUTE_KEYS.R_SCRIPT_NEW });
-                                                    }}
-                                                >
-                                                    <Translate msg="scripts.overview.header.add_button" />
-                                                </Button>
+                                                <Box flex="0 0 auto" mr="8px" width="250px">
+                                                    <TransformDocumentationDialogScript
+                                                        open={this.state.loadDocDialogOpen}
+                                                        onOpen={this.onLoadDocDialogOpen}
+                                                        onClose={this.onLoadDocDialogClose}
+                                                    />
```

Comment:
Check if the dialog is open before rendering it. This way, the dialog is not really mounted if it is not opened.